core/controllers/galleries_test.py

```diff
@@ -125,7 +125,13 @@ def test_gallery_handler_for_created_explorations(self):
             'user_email': self.ADMIN_EMAIL,
             'username': self.ADMIN_USERNAME,
             'search_cursor': None,
-            'profile_picture_data_url': None,
+            'profile_picture_data_url':(
```

Comment:
As `profile_picture_data_url` is now not None , we'll have to compare it with the data uri of the gravatar. I hardcoded the expected result because I think importing files to do hash and then `urlfetch` would become tedious. If there's a better approach to this , please mention